res/res_pjsip_authenticator_digest.c

```diff
@@ -177,7 +177,7 @@ static pj_status_t digest_lookup(pj_pool_t *pool, const pj_str_t *realm,
 		break;
 	case AST_SIP_AUTH_TYPE_MD5:
 		pj_strdup2(pool, &info->data, auth->md5_creds);
-		info->data_type = PJSIP_CRED_DATA_DIGEST;
+		info->algorithm_type = PJSIP_AUTH_ALGORITHM_MD5;
```

Comment:
The patch didn't apply correctly.  It should have just added the `info->algorithm_type` line leaving the `info->data_type` line alone.

Reply:
Yes, I see. I will fix it.

Reply:
Fixed inlocal repo - https://github.com/asterisk/asterisk/compare/master...silentindark:asterisk-master:master

Reply:
@silentindark  You changed the branch name so I can no longer re-open this pull request.  You have to open a new PR.